images/bazelbuild/runner

```diff
@@ -88,24 +85,35 @@ if [[ "${DOCKER_IN_DOCKER_ENABLED}" == "true" ]]; then
         fi
     done
     cleanup_dind
-    printf '=%.0s' {1..80}; echo
-    echo "Done setting up docker in docker."
 fi
 
-# disable error exit so we can run post-command cleanup
+# Disable error exit so we can run post-command cleanup.
 set +o errexit
-# actually start bootstrap and the job
-"$@"
+
+# Run the actual job.
+"$@" &
+
+# Bash does not "trikle down" signals. If the Bash script receives SIGINT
+# and that the above command "$@" is running, then SIGINT won't be passed
+# down to the "$@" command. To work around that, we trap SIGINT and pass it
+# down explicitely.
+#
+# shellcheck disable=SC2064
+trap "kill INT $!" INT
+wait $!
```

Comment:
This is the fix.

Reply:
why is only SIGINT passed and not SIGTERM?

Reply:
That is a good question, I don't know why I restricted myself to SIGINT. We should also support SIGTERM since SIGTERM is probably what is being sent to processes when the Google Cloud VM is preempted.

Reply:
I have added SIGTERM in b14f5947346ce1b1934a1bc518a7c46b5ab52a38.